Anlab.Jobs.MoneyMovement/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Anlab.Core.Data;
+using Anlab.Core.Domain;
+using Anlab.Core.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Anlab.Jobs.MoneyMovement
+{
+    class Program
+    {
+        public static IConfigurationRoot Configuration { get; set; }
+        public static IServiceProvider Provider { get; set; }
+
+        static void Main(string[] args)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json");
+
+
+            var environmentName = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+
+            if (string.Equals(environmentName, "development", StringComparison.OrdinalIgnoreCase))
+            {
+                builder.AddUserSecrets<Program>();
+            }
+
+
+            builder.AddEnvironmentVariables();
+            Configuration = builder.Build();
+            
+
+            IServiceCollection services = new ServiceCollection();
+            services.AddDbContext<ApplicationDbContext>(options =>
+                    options.UseSqlite("Data Source=..\\..\\..\\..\\Anlab.Mvc\\bin\\Debug\\netcoreapp1.1\\anlab.db")

```

Comment:
@jSylvestre Guess this is ok for now but it'll break when we hit 2.0 soon.  Maybe it would be better to make a /data folder or the old App_Data could also work, and have it live there